target/linux/ramips/image/mt7621.mk

```diff
@@ -732,6 +732,17 @@ define Device/iodata_wnpr2600g
 endef
 TARGET_DEVICES += iodata_wnpr2600g
 
+define Device/iptime_a3004ns-dual
+  $(Device/dsa-migration)
```

Comment:
This should be dropped. Your device wasn't supported before DSA migration.

Reply:
Thanks, I'll remove it.

Reply:
My bad, please revert it. Adrian tells me why here: https://github.com/openwrt/openwrt/pull/4416#discussion_r691544161

Reply:
@CHKDSK88 Did it..